packages/snaps-jest/src/internals/request.ts

```diff
@@ -60,6 +72,12 @@ export function handleRequest({
   runSaga,
   request: { id = nanoid(), origin = 'https://metamask.io', ...options },
 }: HandleRequestOptions): SnapRequest {
+  const getInterfaceError = () => {
+    throw new Error(
+      'Unable to get the interface from the Snap: The returned interface may be invalid.',
```

Comment:
Not sure if this error makes sense if the entire request failed?

Reply:
`Unable to get the interface from the Snap: The request to the Snap failed.` ?

Reply:
👍